### PR TITLE
Added BUILD live stream event

### PR DESCRIPTION
**Pull request by KrisvanderMast**

### CODE REVIEW

File: _posts/events/2018-02-05---Enter-The-Matrix-Securing-Microsoft-Azure-Assets.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
 layout: single
 title: "2018-02-05 - Enter The Matrix: Securing Microsoft Azure’s assets"
+<<<<<<< HEAD
```
Comment (tomkerkhove) [inline _posts/events/2018-02-05---Enter-The-Matrix-Securing-Microsoft-Azure-Assets.md]: Please fix this

File: _posts/events/2018-05-07---build-2018-live-stream.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+layout: single
+title: "2018-05-07 - BUILD 2018 live stream"
+date: 2018-04-21 13:37:00 +0000
```
Comment (tomkerkhove) [inline _posts/events/2018-05-07---build-2018-live-stream.md]: Let's align this one with the event

File: _posts/events/2018-05-07---build-2018-live-stream.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+categories: ["events"]
+tags: ["Events"]
+alias: ["/events/2018-05-07---build-2018-live-stream"]
+author: Kris van der Mast
+redirect_from:
+ - /events/2018-05-07---build-2018-live-stream.html
+ - /events/2018/05/07/build-2018-live-stream.html
+---
+
+The developers conference we're all been waiting for is happening again in May. We're talking about Microsoft //BUILD of course. As not everyone can attend in person we thought it would be a great idea to partner with our friends of MADN, OWIC and VISUG and have a live stream. Microsoft Belgium opens up the doors for us.
```
Comment (tomkerkhove) [inline _posts/events/2018-05-07---build-2018-live-stream.md]: `MADN, OWIC and VISUG` maybe mention full names or links in case they want to learn more?

**Comment by tomkerkhove** [comment]
Thanks for adding this!

**Comment by maartenba** [comment]
Thanks @KrisvanderMast ! Best make a fresh clone of the entire repo - there's quite some older stuff in this PR. I have done a clean commit instead - https://github.com/AzugBe/azugbe.github.io/commit/baa626eac60cd32318373cdd5507318576458dd3

**Pull request closed without merging**